### ISSUE TITLE
Infer operation name from the query/mutation name for GraphQL requests

### ISSUE THREAD

**Issue by tvvignesh** [Huachao/vscode-restclient#703]
Hi. Write now, rest client supports naming the requests with the `@name` tag which is great but since GraphQL queries/mutations already have the operation name, it will just be an extra duplication to give the same name twice in `.http` files.

Rather, for GraphQL queries/mutations `@name` can be inferred from the query or mutation name for eg. `query signupAccount {}` means the operation name is  `signupAccount`

Maybe, if `@name` is specified along with the operation name, you can override the name of graphql operations in such cases.

**Comment by dzmitry-lahoda** [comment]
@tvvignesh are mutations work at all?

**Comment by tvvignesh** [comment]
Yes. Mutations do work. I use it in my workflow. Though there are some other bugs to be addressed which I have already reported.
